docs/root/intro/version_history.rst

```diff
@@ -12,6 +12,7 @@ Breaking changes:
 Bugfixes:
 
 - android: fix engine startup crash for when admin interface is enabled. (:issue:`#2520 <2520>`)
+- android: fix engine builder crash on Android 6/7 when using stats sinks. (:issue:`#2622 <2622>`)
```

Comment:
that crash was not a part of any of the manually published releases so not sure whether we need to list it as something that changed.